homeassistant/components/sutro/config_flow.py

```diff
@@ -0,0 +1,101 @@
+"""Adds config flow for Sutro."""
+from __future__ import annotations
+
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.config_entries import ConfigEntry, OptionsFlow
+from homeassistant.core import callback
+from homeassistant.data_entry_flow import FlowResult
+from homeassistant.helpers.aiohttp_client import async_create_clientsession
+
+from .api import SutroApiClient
+from .const import CONF_TOKEN, DOMAIN, PLATFORMS
+
+
+class SutroFlowHandler(config_entries.ConfigFlow, domain=DOMAIN):
+    """Config flow for sutro."""
+
+    VERSION = 1
+    CONNECTION_CLASS = config_entries.CONN_CLASS_CLOUD_POLL
+
+    def __init__(self):
+        """Initialize."""
+        self._errors = {}
+
+    async def async_step_user(self, user_input=None) -> FlowResult:
+        """Handle a flow initialized by the user."""
+        self._errors = {}
```

Comment:
This is not used at class level?

```suggestion"
        errors = {}
```